drivers/Makefile.dep

```diff
@@ -0,0 +1,131 @@
+# driver dependencies (in alphabetical order)
+
+ifneq (,$(filter adt7310,$(USEMODULE)))
+  USEMODULE += periph_gpio
+  USEMODULE += periph_spi
+endif
+
+
+ifneq (,$(filter at86rf2%,$(USEMODULE)))
+  USEMODULE += at86rf2xx
+  USEMODULE += ieee802154
+  USEMODULE += periph_gpio
+  USEMODULE += periph_spi
+  ifneq (,$(filter periph_cpuid,$(FEATURES_PROVIDED)))
+    USEMODULE += periph_cpuid
+  endif
+endif
+
+ifneq (,$(filter dht,$(USEMODULE)))
+  USEMODULE += periph_gpio
+endif
+
+ifneq (,$(filter hdc1000,$(USEMODULE)))
+  USEMODULE += periph_i2c
+endif
+
+ifneq (,$(filter ina220,$(USEMODULE)))
+  USEMODULE += periph_i2c
+endif
+
+ifneq (,$(filter isl29020,$(USEMODULE)))
+  USEMODULE += periph_i2c
+endif
+
+ifneq (,$(filter isl29125,$(USEMODULE)))
+  USEMODULE += periph_gpio
```

Comment:
This driver is based on i2c. Don't you need to add the `USEMODULE += periph_i2c` line here?